applogic/src/api/user_cubit/mod.rs

```diff
@@ -34,6 +35,8 @@ use super::{
 mod qs;
 mod user_handle;
 
+const DELETE_ACCOUNT_CONFIRMATION_TEXT: &str = "delete";
```

Comment:
This is not great for localization of course. Good enough for now, but we should see how we can get l10n in Rust as well.